src/lib/openapi/spec/project-schema.ts

```diff
@@ -89,6 +89,71 @@ export const projectSchema = {
             description:
                 'The average time from when a feature was created to when it was enabled in the "production" environment during the current window',
         },
+        owners: {
+            description: 'The owners of this project, both users and groups.',
+            oneOf: [
+                {
+                    type: 'array',
```

Comment:
We must always have at least one item, right?

```suggestion
                    type: 'array',
                    minItems: 1,
```